### PR TITLE
update cds get started section

### PR DESCRIPTION
**Pull request by silvashenol20**
went through the cds documentation space suggested changes https://docs.google.com/document/d/19LbosY7uhnRyGOugJc8ue-F_UaL86dT6b7P3E4QEWzU/edit#

### CODE REVIEW

**Comment by silvashenol20** [comment]
updated CDS about this release page

File: en/docs/get-started/about-this-release.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
-WSO2 Open Banking UK Toolkit 1.0 provides the technology requirements that banks need to digitally transform and become 
-regulatory-compliant with Payment Service Directive 2 (PSD2) and Open Banking Standard UK v3.1.6 specification.
+WSO2 Open Banking CDR Toolkit 1.0 provides the technology requirements that banks need to digitally transform and become
```
Comment (isharailanga) [inline en/docs/get-started/about-this-release.md]: Tookit name is CDS. Pls change to `CDS`

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
-
-- Data Publishing Executor.
-- Error Handling for the Dynamic Client Registration (DCR) API.
-- Error Handling for Account and Transaction API.
-- Error Handling for Payment Initiation API.
-- Error Handling for Confirmation of Funds API.  
-- Error Handling for Schema Validation.
-- Handling Idempotency for the Payment Initiation Flow.
-- Official Swagger and in-sequence files for the DCR, Account, Payment, and Confirmation of Funds APIs
+####WSO2 Open Banking Identity Server CDS Toolkit
```
Comment (isharailanga) [inline en/docs/get-started/about-this-release.md:16]: Lets remove the full stop at each bullet point as these are not complete sentences. wdyt?

File: en/docs/get-started/about-this-release.md [hunk 3]
```diff
@@ -1,42 +1,40 @@
...
+
+####WSO2 Open Banking API Manager CDS Toolkit
+
+- Support for Consumer Data Standard (CDS) API v1.8.0
+- Consumer Data Standard (CDS) Error Format.
+- Dynamic Client Registration.
+- Consumer Data Standard Specific Header Validations.
+- ID Permanence.
+- Consumer Data Standard Metrics API.
+- Consumer Data Standard (CDS) error format.
```
Comment (isharailanga) [inline en/docs/get-started/about-this-release.md]: Consumer Data Standard (CDS) error format. is duplicated in Line 30 and 35

File: en/docs/get-started/open-banking-requirements.md [hunk 4]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
+   to consume a defined data holder's consumer data set.
 
-   - **Consent grant**: The bank redirects the consent request to the customer to approve/deny.
+   - **Consent grant**: The act of establishing permission by a consumer to a data holder to authorize access to the 
+     consumer data by accredited data recipients.
 
-   - **Consent verification**: The bank verifies if the customer has approved the Data Recipient application to access 
-   the information. If the bank customer has denied the consent, the bank must detect and stop the application from 
-   invoking the banking APIs.
+   - **Consent verification**: The data holder verifies whether the consumer has approved the accredited data recipient
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Shall we use
- Data Holder
- Accredited Data Recipient

File: en/docs/get-started/open-banking-requirements.md [hunk 5]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
 
-   - **Consent grant**: The bank redirects the consent request to the customer to approve/deny.
+   - **Consent grant**: The act of establishing permission by a consumer to a data holder to authorize access to the 
+     consumer data by accredited data recipients.
 
-   - **Consent verification**: The bank verifies if the customer has approved the Data Recipient application to access 
-   the information. If the bank customer has denied the consent, the bank must detect and stop the application from 
-   invoking the banking APIs.
+   - **Consent verification**: The data holder verifies whether the consumer has approved the accredited data recipient 
+     to access the information. If the bank consumer has rejected the consent, the bank must detect and stop the
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Let's use `If the bank customer `

File: en/docs/get-started/open-banking-requirements.md [hunk 6]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
 
-   - **Consent revocation**: A customer can revoke the consent via consent management applications. It can either be 
-   done by the customer themselves or by a bank representative upon the customer’s request.
+   - **Consent revocation**: The act of ending or removing permission for access by the consumer via consent management 
+     applications or by a bank representative upon the customer’s request.
 
-   - **Consent expiration**: When the consent validity period expires, the bank sets the consent status as expired. 
-   For the Data Recipient application to access the customer’s financial information again, the customer needs to regrant 
-   the consent.
+   - **Consent expiration**: An inactive state caused by the occurrence of a time limit and was not renewed.
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Shall we keep the existing description?

File: en/docs/get-started/open-banking-requirements.md [hunk 7]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
-   - **Consent verification**: The bank verifies if the customer has approved the Data Recipient application to access 
-   the information. If the bank customer has denied the consent, the bank must detect and stop the application from 
-   invoking the banking APIs.
+   - **Consent verification**: The data holder verifies whether the consumer has approved the accredited data recipient 
+     to access the information. If the bank consumer has rejected the consent, the bank must detect and stop the 
+     application from invoking the banking APIs.
 
-   - **Consent revocation**: A customer can revoke the consent via consent management applications. It can either be 
-   done by the customer themselves or by a bank representative upon the customer’s request.
+   - **Consent revocation**: The act of ending or removing permission for access by the consumer via consent management
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Shall we keep the existing description?

File: en/docs/get-started/open-banking-requirements.md [hunk 8]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
-   - **Consent expiration**: When the consent validity period expires, the bank sets the consent status as expired. 
-   For the Data Recipient application to access the customer’s financial information again, the customer needs to regrant 
-   the consent.
+   - **Consent expiration**: An inactive state caused by the occurrence of a time limit and was not renewed. 
+     Once expired, the bank (data holder) shall no longer share any data with the specified accredited data recipient.
 
-3. **Consumer Authentication** - Consumer authentication is an authentication mechanism with a layered defence. 
-When a user initiates a payment or accesses information via a Data Recipient application, it authenticates the user 
-using the following factors one at a time:
+3. **Consumer Authentication** - Process by which the End User’s access to Financial Account Information is authenticated
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Shall we keep the existing description?

File: en/docs/get-started/open-banking-requirements.md [hunk 9]
```diff
@@ -25,47 +24,46 @@ fully-featured consent management module that:
...
+3. **Consumer Authentication** - Process by which the End User’s access to Financial Account Information is authenticated 
+   by the data holder. This is accomplished via different mechanisms:
    ![authentication factors](../assets/img/get-started/open-banking-requirements/authentication-factors.png)
   
-     - Knowledge	: something the user knows. For example: password.
-     - Possession	: something the user owns. For example: ATM cards.
-     - Inherence	: something the user is. For example: fingerprint.
+     - Knowledge    : something you know – passwords, PINs, code words, etc.
+     - Possession	: something you have – typically smart phones, token devices, etc.
+     - Inherence	: something you are – Biometrics (e.g., fingerprints, facial recognition, iris or retina scans).
```
Comment (isharailanga) [inline en/docs/get-started/open-banking-requirements.md]: Let's remove the e.g. here.

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

File: en/docs/get-started/open-banking.md [hunk 10]
```diff
@@ -1,16 +1,16 @@
 ## What is Open Banking?
 
-Open banking is a concept that enables banks, customers, and third parties to use and benefit more from the vast silos 
-of data held by banks on their customers. Fundamentally, open banking allows bank customers to share their financial 
-data with third parties with their consent. In technical terms, this is done via secure APIs that allow the customers 
-or authorized Third Party Provider applications to access data held by the bank. It is envisioned that upon wide adoption, an 
-ecosystem of APIs would be created with various third-party applications, services and intermediary platforms 
-integrating with banks to reinvent financial services using customer data. 
+Open banking is a concept that enables data holders, customers, and accredited data recipients to use and benefit more
```
Comment (isharailanga) [inline en/docs/get-started/open-banking.md]: Let's not change the content under `## What is Open Banking?`

File: en/docs/get-started/open-banking.md [hunk 11]
```diff
@@ -46,19 +46,20 @@ of them.
...
-   the consent of the consumer. Data Recipients use standard APIs to access consumer data to provide services. <br/> 
-   A Data Recipient needs to be accredited by the ACCC. Once accredited, an Accredited Data Recipient can request banking 
-   data from a Data Holder with the consent of the consumer.
+   * **Consumer** - The end-user who has benefited from CDR, the consumer can give permission to the relevant data holder 
+   to share their financial data or authorize transactions with the accredited data recipients.
 
-   * **Data Holder** -  The Data Holder is the organization that the CDR is applied to. This is an entity such as a bank 
-   that provides and maintains a payment account for a consumer. With the consumer's consent, the Data Holders share 
-   account and transaction data via API endpoints.
+   * **Data Recipient** - A Data Recipient (DR) is an accredited party that can request CDR consumers data from a
```
Comment (isharailanga) [inline en/docs/get-started/open-banking.md]: should be `consumers'`

**Comment by isharailanga** [comment]
For consistency always use the following terms with Capitalization:

- Data Recipient
- Data Holder

**Review by isharailanga — APPROVED** [review]

**Pull request merged**